src/containers/profile/notifications.svelte

```diff
@@ -1 +1,121 @@
-<!--TODO: notification container of profile page-->
+<script>
+  import Button from 'ui/button.svelte';
+  import RadioGroup from 'ui/radio-group.svelte';
+
+  const items = [
+    {
+      id: 1,
+      label: 'Push',
+    },
+    {
+      id: 2,
+      label: 'Off',
+    },
+    {
+      id: 3,
+      label: 'e-mail',
+    },
+  ];
+
+  const selected = {
+    all: {},
+    store: {},
+    volunteering: {},
+    projs: {},
+  };
+
+  const changeAll = (e) => {
+    const item = e.detail;
+
+    selected.all = item;
+    selected.store = item;
+    selected.volunteering = item;
+    selected.projs = item;
+  };
+
+  const changeOne = (name, e) => {
+    selected[name] = e.detail;
+  };
+</script>
+
+<div class="notifications">
+  <header>
+    <div class="dialog notice">
+      <div class="message">
+        <svg src="images/icons/alert-triangle.svg" class="icon" />
+        <p>Push notifications require permission before they can be used.</p>
+      </div>
+      <div class="actions">
+        <Button>prompt for permissions</Button>
+      </div>
+    </div>
+  </header>
+
+  <ul>
+    <li class="section">
```

Comment:
A lot of code repetition for the `<li>`s. Could be wrapped in a `for` with the unique data in the javascript.

Reply:
Agree with this good practice. Just was lazy to do it before and afraid if radio-group is not that good for it. But now changed it and it suits perfectly for `for` wrapper